website/docs/r/databricks_workspace.html.markdown

```diff
@@ -114,6 +116,8 @@ A `custom_parameters` block supports the following:
 
 * `storage_account_sku_name` - (Optional) Storage account SKU name. Possible values include `Standard_LRS`, `Standard_GRS`, `Standard_RAGRS`, `Standard_GZRS`, `Standard_RAGZRS`, `Standard_ZRS`, `Premium_LRS` or `Premium_ZRS`. Defaults to `Standard_GRS`. Changing this forces a new resource to be created.
 
+~> **Note:** In v4.0 of the provider changing the `storage_account_sku_name` parameter will no longer cause a new resource to be created.
```

Comment:
This isn't a breaking change, we can remove `ForceNew` now.

Reply:
Fixed.